src/services/refactors/convertExport.ts

```diff
@@ -134,7 +149,7 @@ namespace ts.refactor {
                     changes.insertNodeAfter(exportingSourceFile, exportNode, factory.createExportDefault(factory.createIdentifier(exportName.text)));
                     break;
                 default:
-                    Debug.assertNever(exportNode, `Unexpected exportNode kind ${(exportNode as ExportToConvert).kind}`);
+                    Debug.assertNever(exportNode as never, `Unexpected exportNode kind ${(exportNode as ExportToConvert).kind}`);

```

Comment:
This defeats the purpose of `assertNever`. Either restructure this whole control flow to be part of the same `switch`, or just change this to be a `Debug.fail`.

Reply:
fixed